utils/ansible/ftest/vars/Rocky8.yml

```diff
@@ -6,39 +6,60 @@ daos_openmpi_module_path: /usr/share/modulefiles/mpi/openmpi-x86_64
 daos_base_deps:
   rpms:
     - bash
+    - capstone
     - gdb
     - glibc
     - jq
     - lbzip2
-    - mercury
+    - numactl
+    - numactl-libs
+    - numatop
     - platform-python
     - python36
     - systemd
     - tree
 
 daos_dev_deps:
   rpms:
+    - argobots-devel
+    - capstone-devel
+    - bzip2-devel
+    - clustershell
     - cmake
     - createrepo
+    - daos-raft-devel
     - dnf-plugins-core
+    - fuse3-devel
+    - gcc-c++
+    - gcc-gfortran
+    - golang
     - libaec-devel
+    - libattr-devel
+    - libcmocka-devel
+    - libevent-devel
+    - libyaml-devel
     - meson
+    - ninja-build
+    - openssl-devel
+    - python3.11
+    - python3.11-devel
     - python3-clustershell
+    - python3-defusedxml
+    - python3-distro
+    - python3-jira
+    - python3-junit_xml
+    - python3-pip
+    - python3-pyelftools
+    - python3-scons
+    - python3-tabulate
+    - python3-wheel
+    - python3-pyxattr
+    - python3-pyyaml
```

Comment:
Are a lot of these new packages needed now then?  python3.11, python3.11-devel and python-wheel might be enough here.

Reply:
Agree with you, this is some reminder of old experiments.

- [ ] Remove useless python related packages